warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -1300,18 +1306,18 @@ protected void distCpDirectory(Path workDir, FileSystem src, FileSystem dest, Co
         // that when copying across clusters.
         //@formatter:off
         DistCpOptions options = new DistCpOptions.Builder(Collections.singletonList(srcPath), destPath)
-            .withLogPath(logPath)
-            .withMapBandwidth(distCpBandwidth)
-            .maxMaps(distCpMaxMaps)
-            .withCopyStrategy(distCpStrategy)
-            .withSyncFolder(true)
-            .preserve(DistCpOptions.FileAttribute.USER)
-            .preserve(DistCpOptions.FileAttribute.GROUP)
-            .preserve(DistCpOptions.FileAttribute.PERMISSION)
-            .preserve(DistCpOptions.FileAttribute.BLOCKSIZE)
-            .preserve(DistCpOptions.FileAttribute.CHECKSUMTYPE)
-            .withBlocking(true)
-            .build();
+                .withLogPath(logPath)
```

Comment:
Did you reformat this intentionally? If so, that's cool. If this was due to an IDE auto-format - I would put it back, otherwise, these lines will keep getting  ping-ponged back and forth between the two styles.

Reply:
Yeah must've been an auto-format, oops